storage/innobase/buf/buf0flu.cc

```diff
@@ -2087,6 +2091,77 @@ bool buf_flush_do_batch(buf_pool_t *buf_pool, buf_flush_t type, ulint min_n,
   return (true);
 }
 
+/** Waits until a flush batch of the given lsn ends
+@param[in]	new_oldest	target oldest_modified_lsn to wait for */
```

Comment:
you should write in the comment that LSN_MAX is a special value to indicate that we are flushing oldest pages from all the buffers.